app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java

```diff
@@ -55,16 +58,26 @@ public void onAttach(Context context) {
     private void initView() {
         binding.txtLanguage.setText(LocaleUtil.getCurrentLanguage(getActivity()));
         binding.languageSettingsContainer.setOnClickListener(v -> showLanguagesDialog());
-        binding.notificationSwitchRow.init(PrefUtil.KEY_NOTIFICATION_SETTING, true);
-        binding.localTimeSwitchRow.init(PrefUtil.KEY_SHOW_LOCAL_TIME, false);
+
+        boolean shouldNotify = DefaultPrefsSchema.get(getContext()).getNotificationFlag(true);
```

Comment:
I wonder using getContext causes memory leaks cuz it just returns the Context this fragment is currently associated with. Should we use application context?

refs. https://github.com/rejasupotaro/kvs-schema/blob/06d9eaf3b7b156d7e86b6115b0d2bb68129af77c/library/src/main/java/com/rejasupotaro/android/kvs/PrefsSchema.java#L12